packages/core/src/render3/interfaces/container.ts

```diff
@@ -130,6 +132,18 @@ export interface LContainer extends Array<any> {
    * doing so creates circular dependency.
    */
   [VIEW_REFS]: unknown[]|null;
+
+  /**
+   * Array of dehydrated views within this container.
+   *
+   * This information is used during the hydration process on the client.
+   * The hydration logic tries to find a matching dehydrated view, "claim" it
+   * and use this information to do further matching. After that this "claimed"
+   * view is removed from the list. The remaining "unclaimed" views are later
+   * on "garbage-collected", i.e. removed from the DOM once the hydration
```

Comment:
This reads a little better.
```suggestion
   * and use this information to do further matching. After that, this "claimed"
   * view is removed from the list. The remaining "unclaimed" views are "garbage-collected" later
   * on, i.e. removed from the DOM once the hydration
```